code/modules/clothing/spacesuits/rig/rig_verbs.dm

```diff
@@ -6,7 +6,7 @@
 	set category = "Hardsuit"
 	set src = usr.contents
 
-	if(wearer && wearer.back == src)
+	if(wearer)
```

Comment:
Is there any particular reason these are losing the second check, instead of getting one for the belt slot in addition?

Reply:
Not really tbh, fixed